### PR TITLE
Adjusts tag values in payload

### LINKED ISSUE #3386: Adjust tag values in mbp payload
Related:
https://github.com/DoSomething/mbc-transactional-email/issues/11

Adjust "tag" value in mbp payload to support multi tags based on Mandrill `send-template` API documentation: https://mandrillapp.com/api/docs/messages.JSON.html#method=send-template

### PR DESCRIPTION
**Pull request by deezone**
Fixes #3386

Adjusts tag payload values in mbp message requests.

Related:
https://github.com/DoSomething/mbc-transactional-email/issues/11

### CODE REVIEW

File: lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module [hunk 1]
```diff
@@ -122,14 +122,14 @@ function dosomething_mbp_get_transactional_payload($origin, $params = NULL) {
...
-      $payload['email_tag'] = 'drupal_campaign_signup';
+      $payload['email_tags'] = 'drupal_campaign_signup';
       break;
       
     case 'campaign_group_signup':
       dosomething_mbp_get_common_campaign_payload($payload, $params);
       $payload['merge_vars']['CAMPAIGN_COPY'] = $params['transactional_email_copy'];
       $payload['email_template'] = 'mb-group-campaign-signup';
-      $payload['email_tag'] = 'drupal_campaign_group_signup';
+      $payload['email_tag'][] = 'drupal_campaign_group_signup';
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module]: This is missing `email_tags`?  Has this all been tested?  Shouldn't this have failed?

**Comment by deezone** [comment]
Nice catch!

Note not "failed" but rather not sent the tags to `campaign_group_signup` requests sent  to Mandrill for tagging the message. Everything would have continued to work but the tags would have been lost specific to `campaign_group_signup` requests. 

Fixed, note the:

```
[email_tags] => Array
(
  [0] => 1194
  [1] => drupal_campaign_group_signup
)
```

in the payload:

```
Array
(
[activity] => campaign_group_signup
[email] => dlee+local-test-201@dosomething.org
[uid] => 1702622
[subscribed] => 1
[event_id] => 1194
[email_tags] => Array
(
  [0] => 1194
  [1] => drupal_campaign_group_signup
)
[mailchimp_grouping_id] => 10637
[mailchimp_group_name] => TheHunt2014
[merge_vars] => Array
(
[FNAME] => First
[CAMPAIGN_TITLE] => The Hunt
[CAMPAIGN_LINK] => http://dev.dosomething.org:8888/volunteer/hunt
[CAMPAIGN_COPY] => "Some HTML ..."
)
[email_template] => mb-group-campaign-signup
[activity_timestamp] => 1415735607
[application_id] => 1
)
```

**Comment by aaronschachter** [comment]
Seems ok... so, merging this shouldn't affect current email functionality?

**Comment by deezone** [comment]
Nope, tags are more for reporting and segmentation on the Mandrill side.

**Comment by aaronschachter** [comment]
:+1:

**Comment by deezone** [comment]
Related:
https://github.com/DoSomething/mbc-transactional-email/pull/18

**Comment by aaronschachter** [comment]
@DeeZone Did you mistakenly close but not merge?

**Pull request merged**